board/safety/safety_subaru.h

```diff
@@ -20,6 +20,17 @@
 const SteeringLimits SUBARU_STEERING_LIMITS      = SUBARU_STEERING_LIMITS_GENERATOR(2047, 50, 70);
 const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = SUBARU_STEERING_LIMITS_GENERATOR(1000, 40, 40);
 
+const SteeringLimits SUBARU_ANGLE_STEERING_LIMITS = {
+  .angle_deg_to_can = 100,
+  .angle_rate_up_lookup = {
+    {0., 15., 15.},
+    {5.,  .8,  .8}
```

Comment:
In the case of wheel speed sensor failure, we made the lower limit not exceed ISO limits at ~75 mph for Ford, does this do that, assuming a standard steering ratio?

Reply:
this is taken from nissan which has the same units. have a good way to test this from data? these car's will also only be in dashcam mode

Reply:
You can use the vehicle model